### DISCUSSION TITLE
Deleted Codespace secrets are reset

### DISCUSSION THREAD

**Discussion by fabianrmendez** [community/community#12725] [Codespaces] [answered]
When adding a codespace secret via Settings -> Codespaces -> New Secret in github and later deleting it, the secret remains available even after stopping and restarting the Codespaces session. If the secret is deleted manually using "unset MY_SECRET", it then later reappears if the session is stopped and reloaded.

**Comment by tanmayeekamath** [comment, marked as answer]
Hey @fabianrmendez, we had a bug that was causing this that we fixed. Please let us know if you're still experiencing issues around this. Thanks for reporting it.
